### DISCUSSION TITLE
Point_Stat: unsupported interpolation method encountered: HIRA(21)

### DISCUSSION THREAD

**Discussion by MReza-89** [dtcenter/METplus#2659] [Configuration] [answered]
Hi there,

I am trying to use the HiRA technique in my Point_Stat config to verify 24-hour accumulated rainfall forecasts against AWS data. However, I got the following error:
`compute_horz_interp() -> unsupported interpolation method encountered: HIRA(21)`

Could you please explain what this error means and suggest how I might resolve it?

FYI, I tried the BILIN method for this verification case, and it worked fine. Additionally, HiRA worked in another verification case for ensemble forecasts (using Ensemble_Stat).

Thanks,
Reza

**Comment by j-opatz** [comment, marked as answer]
Hi Reza,

And thank you for your question. From what you described and what error you're receiving, I think you may be trying to access HiRA interpolation through the `interp` dictionary type, which is only accessible through Ensemble-Stat. This is why when you attempted this method through Ensemble-Stat you were able to successfully complete your verification.

When using Point-Stat, you'll need to utilize the `hira` dictionary instead, which is separate from the `interp` dictionary. Information on that dictionary and its settings are located [here](https://met.readthedocs.io/en/latest/Users_Guide/config_options.html#hira), in the MET User's Guide.

You may also find the [HiRA framework section](https://met.readthedocs.io/en/latest/Users_Guide/point-stat.html#hira-framework) useful, as it goes into the methodology behind the scenes of how a pseudo ensemble is accomplished through point interpolation.

**Comment by MReza-89** [comment]
Hi @j-opatz 

Thank you so much for your answer and for the information. I can confirm that I could run Point_Stat for my verification case using `hira` dictionary. [Here](https://metplus.readthedocs.io/en/latest/Users_Guide/glossary.html#term-POINT_STAT_HIRA_FLAG) is link to information on `hira` settings in the METplus user's guide.

Cheers,
Reza
